newsfragments/2810.bugfix.rst

```diff
@@ -1 +0,0 @@
-Regenerate Ursula TLS certificates if the become invalid, e.g. become expired.
```

Comment:
What's up with the removed newsfragments? Mis-commit?

Reply:
There was some issue with the release the last time - see https://discord.com/channels/411401661714792449/412681039178366996/899655352767889469, where the release note that was supposed to be generated by the newsfragment wasn't included for some reason for the v5.3.2 release, so the newsfragment was never removed. 

That reminds me, I should add the release note for 5.3.2 to the releases docs page as part of this PR, or else it won't be included at all in the releases page i.e. there would only be release notes for 5.3.1 and 5.3.3. I've updated the release page now.